wash_mockapi.cpp

```diff
@@ -167,4 +167,24 @@ namespace wash {
         }
     }
 
+    std::vector<std::string>* sim_get_forces_scalar() {
```

Comment:
Better to return a const reference here e.g. `const &std::vector<std::string>`

Reference means that the calling function cannot change the address (unlike a ptr)

If you make it const it enforces that the calling function cannot change the value of these forces.

Reply:
Changes made.